### PR TITLE
added a provision to run a flask app

### PR DESCRIPTION
**Pull request by Omeramsc**
@lioramilbaum as asked in issue #9

### CODE REVIEW

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Please rebase your PR with the latest changes in upstream (I have merged @Yarinh55 PR)

**Comment by Omeramsc** [comment]
@lioramilbaum  fixed and ready to be pulled

File: .vagrant/rgloader/loader.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
```
Comment (lmilbaum) [inline .vagrant/rgloader/loader.rb]: This file should be ignored. It is  in the vagrant cache folder, and appears in the .gitignore file.

File: Vagrantfile [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "fedora/30-cloud-base"
-  config.vm.provision "shell", path:"ProvisionScripts.sh"
-end
+  config.vm.provider "virtualbox" do |v|
+    v.name = "tiger_dev"
+    v.memory = 2048
```
Comment (lmilbaum) [inline Vagrantfile]: why did you increased the memory? is it required?

File: scripts/bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+dnf -y update
+dnf -y install python3-pip
+pip3 install --user Flask
+
+python3 /vagrant/scripts/app.py
```
Comment (lmilbaum) [inline scripts/bootstrap.sh]: You can omit the python3 command. The shabang in app.py and the execute permissions should take care of it.

File: scripts/app.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
```
Comment (lmilbaum) [inline scripts/app.py]: This is not really a script. It is the application.  Move the file to the project rood folder.

**Review by lmilbaum — CHANGES_REQUESTED** [review]

**Comment by lmilbaum** [comment]
@Omeramsc Before you Re-request a review, please squash your commits into one

File: Vagrantfile (hunk 2 above)
Comment (Omeramsc) [reply Vagrantfile]: Thought we meant for it to be so because of the presentation. fixed.

**Pull request closed without merging**